datalad/interface/utils.py

```diff
@@ -520,7 +522,13 @@ def _process_results(
             if hasattr(cmd_class, 'custom_result_renderer'):
                 cmd_class.custom_result_renderer(res, **kwargs)
         elif hasattr(result_renderer, '__call__'):
-            result_renderer(res, **kwargs)
+            try:
+                result_renderer(res, **kwargs)
+            except Exception as e:
+                lgr.warn('Result rendering failed for: %s [%s]',
```

Comment:
apparently this code is indeed not tested even in master: https://codecov.io/gh/datalad/datalad/src/master/datalad/interface/utils.py#L524